setup.sh

```diff
@@ -22,13 +23,14 @@ usage(){
 	echo "-j [cores]          run CMSSW compilation on # cores (default = ${CORES})"
 	echo "-n [name]           name of the CMSSW directory (default = ${CMSSWVER})"
 	echo "-d [dir]            project installation area for the CMSSW directory (default = ${DIR})"
+	echo "-D                  print additional debug statements (default = ${DEBUG})"
```

Comment:
add to readme?

Reply:
Done. I also added some other undocumented options.